apps/experiments/models.py

```diff
@@ -433,19 +455,28 @@ class ExperimentRoute(BaseTeamModel):
     type = models.CharField(choices=ExperimentRouteType.choices, max_length=64, default=ExperimentRouteType.PROCESSOR)
     condition = models.CharField(max_length=64, blank=True)
 
+    parent_version = models.PositiveIntegerField(null=True, blank=True)
```

Comment:
Are we not also copying these experiment routes? (I can't remember if we said we're going to do it or not)